components/scream/src/control/tests/sc_tests.cpp

```diff
@@ -145,12 +145,263 @@ TEST_CASE ("surface_coupling")
     v3d_imp.sync_to_host();
     for (int icol=0; icol<ncols; ++icol) {
       REQUIRE (s2d_exp_h(icol)==s2d_imp_h(icol));
-      REQUIRE (s3d_exp_h(icol,0)==s3d_imp_h(icol,0));
-      REQUIRE (v3d_exp_h(icol,0,0)==v3d_imp_h(icol,0,0));
+      REQUIRE (s3d_exp_h(icol,nlevs-1)==s3d_imp_h(icol,nlevs-1));
+      REQUIRE (v3d_exp_h(icol,0,0)==v3d_imp_h(icol,0,0)); // Is this correct?
       REQUIRE (v3d_exp_h(icol,0,1)==v3d_imp_h(icol,0,1));
```

Comment:
Yes. I think we discussed this when we last chatted. Throughout the SC code, I accidentally assumed that the surface is at ilev=0. Since I made the mistake both in the SC impl and in the unit test, things worked out. Once you fixed the SC impl, this test needs to be fixed as well.